omod/src/main/webapp/resources/htmlforms/HMIS-TB-003-TBClientCard-EncounterPage.xml

```diff
@@ -0,0 +1,418 @@
+<htmlform formName="HMIS TB 003: TB Client Card Encounter Page" formDescription="Admission into a clinic for a pregnant woman to give birth" formEncounterType="a9f11592-22e7-45fc-904d-dfe24cb1fc67" formUuid="0248cd00-bc53-11ea-b3de-0242ac130004" formVersion="3.0.0" formAddMetadata="yes" formUILocation="patientDashboard.visitActions" formOrder="9" formIcon="icon-medkit" formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false &amp;&amp; patient.person.gender=='F' &amp;&amp;sessionLocation.uuid=='e9bc61b5-69ff-414b-9cf0-0c22d6dfca2b'" formDisplayStyle="Standard" formLabel="HMIS-TB-003-TB CLIENT CARD - ENCOUNTER PAGE">
+
+	<script type="text/javascript">
+		if (jQuery) {
+
+		    jq(document).ready(function () {
+
+				//Block Multiple encounter entry on same date
+				blockEncounterOnSameDateEncounter(jq("#encounterDate").find("input[type=hidden]"),"block");
+
+				// Validate MUAC
+                var age  = <lookup complexExpression="$patient.age" />; 
+                getMUACCodeFromMUACScoreByAge(age, "muac-score", "muac-code");
+
+                disable_fields("other-complications-of-pregnancy");
+                disable_fields("other-management-procedure");
+                disable_fields("other-source-of-warmth");
+                disable_fields("other-ward-or-department");
+                disable_fields("transferred-by");
+                disable_fields("other-mode-of-delivery");
+
+                //enable/disable other management procedure
+                jq("#165443").change(function () {
+	                if(jq(this).find("select").find(":selected").val() == "5622") {
+	                    enable_fields("other-management-procedure");
+                    } else {
+                    	disable_fields("other-management-procedure");
+                    }
+                });
+
+                //enable/disable other source of warmth
+                jq("#165446").change(function () {
+	                if(jq(this).find("select").find(":selected").val() == "90002") {
+	                    enable_fields("other-source-of-warmth");
+                    } else {
+                    	disable_fields("other-source-of-warmth");
+                    }
+                }); 
+			});
+		}
+	</script>
+
+	<div>
+		<div class="row">
+			<div class="col-md-8">
+				<H1>HMIS TB 003: TB Client Card - Encounter Page</H1>
+			</div>
+			<div class="col-md-3"></div>
+			<div>
+				<div class="col-md-1">
+					<submit class="confirm" submitLabel="Save" submitStyle="customSubmit" />
+				</div>
+			</div>
+		</div>
+		<div class="row">
+			<div class="col-md-12">
+				<div class="card">
+					<div class="card-header">
+						<ul class="nav nav-tabs card-header-tabs">
```

Comment:
1. Add section for Treatment Phase and in it add the encounterDate and the Treatment Phase in a [<workflowState>](https://wiki.openmrs.org/display/docs/HTML+Form+Entry+Module+Reference#HTMLFormEntryModuleReference-%3CworkflowState%3E) tag

Reply:
Section added but i dont seem to know how to add Treatment Phase in a tag

Reply:
Something like this
<workflowState workflowId="1" labelText="TB Treatment Phase" stateIds="1,2,4" stateLabels="Initial Phase,Continuation Phase, Retreatment Phase" style="radio" />

Reply:
@slubwama where do i find workflowIds from? Look at this error 'java.lang.IllegalArgumentException: workflow does not exist: 1'